portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,42 @@
+body {
+  margin: 0;
+}
+
+#fullpage {
+  min-width: 100vw;
+  min-height: 100vh;
+  min-width: 100vw;
+  min-height: 100vh;
+  overflow-x: hidden;
+  overflow-y: scroll;
+}
+
+.section {
+  min-height: 100vh;
+  max-height: 100vh;
+  overflow: hidden;
+  min-width: 100%;
+  max-width: 100%;
+  text-align: center;
+  display: flex;
```

Comment:
[alphabetize declarations](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order), here and elsewhere

Reply:
Updated in commit e83f090